test/error_messages/storage.py

```diff
@@ -14,7 +14,7 @@ def check_error(desc, fn, *required_substrings):
         for sub in required_substrings:
             assert sub in error_message
         return
-    assert False, "given function ({}) didn't raise an error".format(desc)
+    assert AssertionError("given function ({}) didn't raise an error".format(desc))
```

Comment:
Umm, are you sure this is the idiomatic spelling? Shouldn't it be `raise AssertionError`?

Reply:
You are right, I've pushed a new commit to correct this.